luigi_pipeline/seqr_gcnv_loading.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import sys
+
+import luigi
+import hail as hl
+
+from lib.model.gcnv_mt_schema import SeqrGCNVVariantSchema, SeqrGCNVGenotypesSchema, SeqrGCNVVariantsAndGenotypesSchema
+from luigi_pipeline.seqr_loading_optimized import SeqrVCFToVariantMTTask, BaseVCFToGenotypesMTTask, BaseMTToESOptimizedTask
+from sv_pipeline.genome.utils.mapping_gene_ids import load_gencode
+
+
+logger = logging.getLogger(__name__)
+
+SAMPLE_ID_REGEX = r'(.+)_v\d+_Exome_(C|RP-)\d+$'
+
+class SeqrGCNVVariantMTTask(SeqrVCFToVariantMTTask):
+    # Overrided inherited required params.
+    reference_ht_path = ""
+    clinvar_ht_path = ""
+    sample_type = "WES"
+    genome_version = "38"
+    dont_validate = True
+    dataset_type = "SV"
+
+    RUN_VEP = False
+    SCHEMA_CLASS = SeqrGCNVVariantSchema
+
+    def annotate_old_and_split_multi_hts(self, mt, *args, **kwargs):
+        return mt
+
+    def add_37_coordinates(self, mt, *args, **kwargs):
+        # TODO.. implement this (it was out of scope in the port but we want it eventually)
```

Comment:
did we make a separate ticket for this? Just want to make sure it doesn't get forgotten

Reply:
yep! https://github.com/broadinstitute/seqr-loading-pipelines/issues/399